ozaria/site/components/teacher-dashboard/modals/ModalLicenseStats.vue

```diff
@@ -0,0 +1,112 @@
+<script>
+  import moment from 'moment'
```

Comment:
why do we need to create this? Can't we use the Coco one?

Reply:
that coco one uses backbone with vue together, is not so clean to use in ozaria. and actually i think we would replace coco's teacher-dashboard with ozaria one later right? so i think make a ozaria version and later replace the coco one would be better 😂